pod/pod-api-public.yaml

```diff
@@ -5428,6 +5428,13 @@ definitions:
         enum:
           - ENABLED
           - DISABLED
+      suspended:
```

Comment:
should not we provide descriptions for each field? (at least for the suspendedUntil to explain it is is a TS since...)
by the way is the format consistent with date fields returned in this WS payload?